compose/foundation/foundation/src/jsMain/kotlin/androidx/compose/foundation/internal/ClipboardUtils.js.kt

```diff
@@ -58,10 +58,10 @@ internal actual fun ClipEntry?.hasText(): Boolean {
 }
 
 internal actual fun Clipboard.isReadSupported(): Boolean =
-    js("window.navigator.clipboard && window.navigator.clipboard.read")
+    js("Boolean(window.navigator.clipboard && window.navigator.clipboard.read)")
 
 internal actual fun Clipboard.isWriteSupported(): Boolean =
-    js("window.navigator.clipboard && (window.navigator.clipboard.write || window.navigator.clipboard.writeText)")
+    js("Boolean(window.navigator.clipboard && (window.navigator.clipboard.write || window.navigator.clipboard.writeText))")
```

Comment:
this is js target - why we need this js wrapper and not acessing this entities directly? They are not declared in the stdlib?